privacyIDEA-impl/src/main/resources/META-INF/net/shibboleth/idp/flows/authn/privacyIDEA2/plugin-flow.xml

```diff
@@ -0,0 +1,103 @@
+<flow xmlns="http://www.springframework.org/schema/webflow" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="http://www.springframework.org/schema/webflow http://www.springframework.org/schema/webflow/spring-webflow.xsd"
+      parent="authn.abstract, authn/conditions">
+
+    <!-- This is a login flow for multifactor authentication against the privacyIDEA -->
```

Comment:
This is a login flow for multifactor authentication with privacyIDEA

Reply:
[a2e5c32](https://github.com/privacyidea/shibboleth-plugin/pull/71/commits/a2e5c32b713be96d48d8ed97a855a318380cab1a)